docs/DeveloperGuide.adoc

```diff
@@ -426,32 +426,44 @@ image::LoanIdAddCommandSequenceDiagram.png[width="800"]
 // end::loanid[]
 
 // tag::adminauthentication[]
-=== Admin Authentication
-Before critical actions such as deleting a loan can be performed, admin authentication
-should be required. This ensures that only authorized users are able to perform critical actions.
+=== Authentication
+Before critical actions such as deleting a loan can be performed, authentication of user
+is required. This ensures that only authorized users are able to perform critical actions.
 This is done by requiring a password before performing a critical action.
 
 ==== Current Implementation
-The authentication mechanism is provided by `Password`. It implements the following operations that facilitates authentications:
+|===
+| _The following provide a walkthrough on how the authentication feature is implemented._
+|===
+The authentication mechanism is provided by the `Password` class. It implements the following operations that facilitates authentications:
 
 * `Password#isSamePassword(currentPass, providedPass)` -- Checks if both `providedPass` and `currentPass` are the same password after decryption.
-* `Password#generateSalt()` -- Generate a hash salt for the app.
+* `Password#generateSalt()` -- Generate a hashing salt for the app.
 * `Password#encrypt(providedPass, salt)` -- Encrypt `providedPass` using hashing `salt`.
 
 Given below is an example usage scenario and how the authentication mechanism behaves at each step:
 
-Step 1. The user launches the application for the first time. `UserPref` will be initialised with a randomly generated salt by calling `Password#generateSalt()`, and a default hashed password from `a12345`.
+*Step 1*. The user launches the application for the first time. `UserPref` will be initialised with a randomly generated salt by calling `Password#generateSalt()`, and a default hashed password from `a12345`.
+
+Example of a hash salt is:
```

Comment:
You changed the previous term to "hashing salt" but still use "hash salt" here

Reply:
Good catch.